documentation/source/newsfragments/1943.issue.rst

```diff
@@ -0,0 +1,11 @@
+**Cocotb must be installed through conda when using more than one conda environment**
+
+Affected users: Users of conda with more than one conda environment.
+
+Problem: When using ``pip install cocotb`` in a conda environment, binary packages are produced which are tied to this specific environment.
+After switching to another conda environment, incompatible packages are reused.
```

Comment:
I thought the case was more uninteresting than this, and the problem only arose if the original conda environment was deleted.

Reply:
This is correct.